logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationGroupRepository.kt

```diff
@@ -214,4 +216,11 @@ internal class ConversationGroupRepositoryImpl(
             it.data?.let { data -> conversationDAO.updateGuestRoomLink(conversationId.toDao(), data.uri) }
             it.uri?.let { link -> conversationDAO.updateGuestRoomLink(conversationId.toDao(), link) }
         }.map { Either.Right(Unit) }
+
+    override suspend fun revokeGuestRoomLink(conversationId: ConversationId): Either<NetworkFailure, Unit> =
+        wrapApiRequest {
+            conversationApi.revokeGuestRoomLink(conversationId.toApi())
+        }.onSuccess {
+            conversationDAO.updateGuestRoomLink(conversationId.toDao(), "")
```

Comment:
is is initial to update the the guest link with empty string ?

Reply:
yes, if it's empty then we display the create link button

Reply:
woudln't updating the link with null makes more sense if it is missing, just to avoid checking if the link is empty every time the link need to me used

Reply:
It's same thing. I don't like using nullable value in UI

Reply:
From my perspective it is not the same, seeing that a model/function have a nullable value will force consumers to handle the case of null and indicate clearly the value not existing. On the other hand, an empty string will definitely go unnoticed every so often.